storage/bbolt/dedup.go

```diff
@@ -184,3 +193,24 @@ func itob(idx uint64) []byte {
 func btoi(b []byte) uint64 {
 	return binary.BigEndian.Uint64(b)
 }
+
+// vtob concatenates an index and timestamp values into a byte array.
+func vtob(idx uint64, timestamp uint64) ([]byte, error) {
+	i := itob(idx)
+	if len(i) != 8 {
+		return nil, fmt.Errorf("input error, idx should be %d bytes long, got %d", 8, len(i))
+	}
+	t := itob(timestamp)
+	if len(t) != 8 {
+		return nil, fmt.Errorf("input error, idx should be %d bytes long, got %d", 8, len(t))
+	}
+	return append(itob(idx), itob(timestamp)...), nil
```

Comment:
Could you use `binary.Append` or `binary.Write` to do this, or perhaps `gob` and encode the SCTInfo struct in one piece, rather than doing this by hand?

Reply:
I did try using gob, but the returned structure was larger, I switched to binary.Append / Decode.